atlp-rwanda/knights-ecomm-fe
# Implement Profile Page for Vendor and Admin

#### What does this PR do?
This pull request implement profile page for both vendor and admin

#### Description of Task to be completed?
- Implement Profile page Component for both Vendor and Admin
- User can change profile picture
- User can enable or disable 2FA(two factor authentication)
- User can change his/her basic information

#### How should this be manually tested?
- Ensure You have backend service running and set VITE_APP_API_URL in your .env file.
- Clone this repository and navigate to the project directory
    ```
    cd knights-ecomm-fe    
    git fetch
    git checkout ft-dashboard-account
    npm i
    npm run dev
    ```
try to change profile information

UI Page
![image](https://github.com/user-attachments/assets/efbb002a-feaf-472b-b1b0-2f2037c025ea)
